packages/lib/.size-limit.cjs

```diff
@@ -32,7 +32,7 @@ module.exports = [
     {
         name: 'UMD',
         path: 'dist/umd/adyen.js',
-        limit: '116 KB',
+        limit: '120 KB',
```

Comment:
[Question] do we have a target size limit that we plan not to ever exceed for the bundle sizes?